server/utils/contentNegotiation.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+const js2xmlparser = require("js2xmlparser");
+const json2html = require('json-to-html');
+const {toPlainText} = require('json-to-plain-text');
+
+class ContentNegotiation{
+    constructor(res, statuscode, data) {
```

Comment:
```suggestion
    constructor(res, statusCode, data) {
```
status code is two words. use camelcase for multiple words

Reply:
ok